ui/app/components/splash-page.js

```diff
@@ -7,6 +7,7 @@ export default Component.extend({
   auth: service(),
   store: service(),
   tagName: '',
+  showTruncatedNavBar: true,
```

Comment:
we have the one case where we are using the splash page component but need to show the navbar hence the new conditional.

(I have a todo in another PR to glimmerize the splash page component and add documentation. However, it's outside the scope of this already large PR).